mitmproxy/addons/view.py

```diff
@@ -183,6 +183,10 @@ def load(self, loader):
             "console_focus_follow", bool, False,
             "Focus follows new flows."
         )
+        loader.add_option(
+            "filter_active", bool, False,
```

Comment:
Let's call this `view_filter_active` please so that it's symmetric with `intercept` and `intercept_active`.

Reply:
Ok, that makes sense. I'll wait until we've confirmed that we want this feature before changing this.